static-site/content/projects.jp.md

```diff
@@ -48,4 +48,10 @@ name = "Unifont-rs"
 url = "https://github.com/mkovaxx/unifont-rs"
 logo_url = ""
 description = "Unifont provides a monochrome bitmap font that covers the entire Unicode Basic Multilingual Plane. Halfwidth glyphs are 8x16, fullwidth are 16x16 pixels. Supports #[no_std] builds."
+
+[[extra.project]]
+name = "Fungi"
+url = "https://github.com/enbop/fungi"
+logo_url = "https://raw.githubusercontent.com/enbop/fungi/6d5d791090f529e4fbb0e4d0238e99084b502aa4/assets/FullLogo_Transparent_NoBuffer.png"
```

Comment:
@Heap-Hop If you pin the content to a commit, the logo here won't follow updates you make. So maybe it's better not to pin to a commit?